lib/matplotlib/text.py

```diff
@@ -309,6 +315,27 @@ def get_rotation_mode(self):
         """Return the text rotation mode."""
         return self._rotation_mode
 
+    def set_antialiased(self, b):
+        """
+        Set whether to use antialiased rendering.
+
+        Parameters
+        ----------
+        b : bool
+
+        Notes
+        -----
+        Antialiasing will be determined by ``rcParams['text.antialiased']``
```

Comment:
```suggestion
        Antialiasing will be determined by :rc:`text.antialiased`
```
This will give a better doc-string.